lib/tokenize.cpp

```diff
@@ -1742,6 +1742,34 @@ void Tokenizer::createTokens(const simplecpp::TokenList *tokenList)
     list.createTokens(tokenList);
 }
 
+void Tokenizer::checkOperators() const
+{
+    for (const Token *tok = list.front(); tok; tok = tok->next()) {
+        if(Token::Match(tok, "%or%|%oror%|%assign%|%comp%")) {
```

Comment:
I want that you start running the `runastyle` script. Did you have some problem compiling it?

Reply:
If you search the `git log` for `astyle formatting` you can see that I have to run it myself quite often after I have merged your changes.. I do not like this.

Reply:
The problem was installing it. It was trying to install to `/usr` even though I set the installation path to somewhere else. It looks like its fixed on the latest trunk of astyle. If I download from [here](https://sourceforge.net/code-snapshots/svn/a/as/astyle/code/astyle-code-r655-trunk-AStyle.zip), it does install correctly.

Reply:
Well calling `runastyle` doesn't work, it says:

```
$ ./runastyle
You should use: Artistic Style Version 3.0.1
Detected: Artistic Style Version 3.2 beta
```

Reply:
So this `.clang-format` file is close to what you run for astyle:

```
BasedOnStyle: LLVM
IndentWidth: 4
UseTab: Never
ColumnLimit: 0
BreakBeforeBraces: Custom
BraceWrapping:
  AfterClass: false
  AfterControlStatement: false
  AfterEnum: false
  AfterFunction: true
  AfterNamespace: false
  AfterStruct: false
  AfterUnion: false
  AfterExternBlock: false
  BeforeCatch: false
  BeforeElse: false
  IndentBraces: false
  SplitEmptyRecord: false
  SplitEmptyNamespace: false
  SplitEmptyFunction: false
AllowShortIfStatementsOnASingleLine: false
IndentCaseLabels: false
SpaceBeforeParens: ControlStatements
```

 I think I will just run that for formatting for now.

Reply:
I do not run "make install".. I just copy the "astyle" binary to a folder in my path. then it works. not 100% clean but that works also.